slack-lint/src/main/java/slack/lint/DeprecatedAnnotationDetector.kt

```diff
@@ -36,14 +42,37 @@ class DeprecatedAnnotationDetector : AnnotatedClassOrMethodUsageDetector() {
   override val isEnabled: Boolean
     get() = !LintClient.isStudio
 
-  companion object {
-    private const val DEPRECATED_ANNOTATION_NAME_JAVA = "java.lang.Deprecated"
-    private const val DEPRECATED_ANNOTATION_NAME_KOTLIN = "kotlin.Deprecated"
+  override fun visitAnnotationUsage(context: JavaContext,
```

Comment:
Let's indent these all one over to the left by moving the first one on to its own line